src/network/networkprotocol.h

```diff
@@ -965,4 +974,12 @@ const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
 	"This server has experienced an internal error. You will now be disconnected."
 };
 
+enum PlayerListModifer
```

Comment:
we are using C++11 we can do a nice thing: enum PlayerListModifier: uint8_t  

This will guarantee enum is a uin8_t here instead of int